### ISSUE TITLE
`<Validate>` not working 😢 

### ISSUE THREAD

**Issue by fddayan** [kbrueckner/mui-validate#10]
I really like this package but it does not seem to be working 

this is a snipped of my code. There is text in the field but it shows and error message. `<AutoDisabler>` also disabled the button even if there is text in the text field. 

```
 <ValidationGroup>
            <Grid container spacing={2}>
              {step === 1 && (
                <>
                  <Grid item xs={12} sm={6}>
                    <Validate name="firstNameRequired" required>
                      <TextField
                        error={error}
                        autoComplete="given-name"
                        name="firstName"
                        required
                        fullWidth
                        id="firstName"
                        label="First Name"
                        autoFocus
                        defaultValue={user.firstName}
                        onChange={handleOnChange('firstName')}
                      />
                    </Validate>
                  </Grid>
......
```

![image](https://user-images.githubusercontent.com/188936/165173535-458cb713-3f58-4299-9b17-f0c3fc5a504f.png)

**Comment by kbrueckner** [comment]
I think your problem might go back to minor issue in the auto detection of the underlaying input type.
Please try try set inputType="textfield" on Validate and let me know if that fixed your problem.

If that is the case I can provide a fix for that lookup problem with the next version.

**Comment by kbrueckner** [comment]
A fix for the mentioned issue is released with version 0.1.1
